src/bots/botRouter.ts

```diff
@@ -46,9 +47,9 @@ export class BotRouter implements IBotRouter {
    *
    * @beta
    */
-  constructor (bots: IBot[]) {
+  constructor (bots: Bot[]) {
     if(bots.length === 0) {
-      throw new Error('Error: parameter "bots: IBot[]" cannot be empty as there will be no bots to route')
+      throw new Error('Error: parameter "bots: Bot[]" cannot be empty as there will be no bots to route')
     }
     this.botMap = bots.reduce((acc, bot) => {
       bot.urls.forEach(url => {
```

Comment:
Does this code work for multiple `Bots` with the same url? Say I have two or more `FreeCrunchbase` bots, does this code work or only end up with one but being registered for that url? Is there a test for this behavior, if not we need to create one.

I would have expected the code to be `acc[url].push(bot)`